{
  "id": "CVE-2025-43014",
  "sourceIdentifier": "cve@jetbrains.com",
  "published": "2025-04-17T16:16:00.037",
  "lastModified": "2025-04-17T20:21:05.203",
  "vulnStatus": "Awaiting Analysis",
  "descriptions": [
    {
      "lang": "en",
      "value": "In JetBrains Toolbox App before 2.6 the SSH plugin established connections without sufficient user confirmation"
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "cve@jetbrains.com",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "NONE",
          "userInteraction": "REQUIRED",
          "scope": "CHANGED",
          "confidentialityImpact": "LOW",
          "integrityImpact": "LOW",
          "availabilityImpact": "NONE",
          "baseScore": 6.1,
          "baseSeverity": "MEDIUM"
        },
        "exploitabilityScore": 2.8,
        "impactScore": 2.7
      }
    ]
  },
  "weaknesses": [
    {
      "source": "cve@jetbrains.com",
      "type": "Primary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-304"
        }
      ]
    },
    {
      "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
      "type": "Primary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-304"
        }
      ]
    }
  ],
  "references": [
    {
      "url": "https://www.jetbrains.com/privacy-security/issues-fixed/",
      "source": "cve@jetbrains.com"
    }
  ]
}